src/hooks/SymbolOfERC20.ts

```diff
@@ -1,22 +1,30 @@
 import { useErc20Contract } from 'hooks/Erc20Contract'
 import { useEffect, useState } from 'react'
+import * as constants from '@ethersproject/constants'
 
 /** Returns symbol for ERC20 token with `address`. */
-export default function useSymbolOfERC20(address: string | undefined) {
+export default function useSymbolOfERC20(tokenAddress: string | undefined) {
   const [data, setData] = useState<string>()
 
-  const contract = useErc20Contract(address)
+  const contract = useErc20Contract(tokenAddress)
```

Comment:
The issue was that: if `tokenAddress` was 0, `contract` wouldn't change. Thus, contract.symbol() would return the old token's symbol.

Code below ensures we account for the Zero case in this hook.

Reply:
Beautiful

Reply:
Beautiful